### PR TITLE
Qmaps 872: 4 new categories

### PR DESCRIPTION
**Pull request by xem**
## Description
*Add 4 PoI categories in the services menu and in the POI "anywhere" panel (search around this place)*

## Why
*We want 12 categories*

## Screenshots
image

NB: the fitness category is labeled as "sport" and uses the "basket ball" icon from the map iconfont.
Later, we will replace the icon with the one made by m.chessa (man lifting a weight)

### CODE REVIEW

File: src/views/poi_panel.dot [hunk 1]
```diff
@@ -119,7 +119,7 @@
             {{~ this.categories:category:index }}
               <button class="service_panel__category" type="button" {{= click(this.openCategory, this, category) }}>
                 <div class="service_panel__category__icon" style="background: {{= category.backgroundColor }}">
-                  <span class="icon icon-{{= category.iconName }}"></span>
+                  <span class="{{? index < 9 || index == 10 }}icon{{??}}category__new-icon{{?}} icon-{{= category.iconName }}"></span>
```
Comment (bbecquet) [inline src/views/poi_panel.dot]: Not sure I understand this rule… Why is it using a new CSS class?

File: src/views/service_panel.dot [hunk 2]
```diff
@@ -14,12 +14,12 @@
     {{~ this.categories:category:index }}
       {{?
         !this.mustDeployCategories ||
-        (this.mustDeployCategories && !this.isDeployed && index + 1 < 9) ||
+        (this.mustDeployCategories && !this.isDeployed && index < 12) ||
```
Comment (bbecquet) [inline src/views/service_panel.dot]: I recently found that the whole "deployment" logic is dead code. `isDeployed` is changed only in the `toggleCategories` function, which is never called anywhere. Maybe we can get rid of all that.

File: src/views/poi_panel.dot (hunk 1 above)
Comment (xem) [reply src/views/poi_panel.dot]: Yeah, it's a bit dirty for now, as we discussed on the map Thread last week.
9 of the icons are present in the icon font shared with the map (class=icon)
3 new icons are only present in the icomoon font (class=new_icon)
It's agreed to merge like that for now, and add the 3 new icons in the shared font when we have the time (Adrien needs to teach us how to do it)

File: src/views/service_panel.dot (hunk 2 above)
Comment (xem) [reply src/views/service_panel.dot]: ok!

Comment (xem) [reply src/views/service_panel.dot]: The whole "deploy" thing was used in an older design, and doesn't exist anymore. I removed it

File: src/views/poi_panel.dot (hunk 1 above)
Comment (bbecquet) [reply src/views/poi_panel.dot]: Alright! Thanks for the explanation :)

File: src/views/service_panel.dot [hunk 3]
```diff
@@ -12,11 +12,7 @@
   {{? this.categories.length > 0 }}
   <div class="service_panel__categories">
     {{~ this.categories:category:index }}
-      {{?
-        !this.mustDeployCategories ||
-        (this.mustDeployCategories && !this.isDeployed && index < 12) ||
-        (this.mustDeployCategories && this.isDeployed)
-      }}
+      {{? index < 12 }}
```
Comment (bbecquet) [inline src/views/service_panel.dot]: I guess you could even remove this test for now, but it's no big deal :)

Comment (xem) [reply src/views/service_panel.dot]: consider it a "garde-fous" in case someone touches categories.yml

**Pull request merged**